sqlobject/src/main/java/org/jdbi/v3/sqlobject/CustomizingStatementHandler.java

```diff
@@ -85,8 +85,8 @@
                     // we have a binder
                     BindingAnnotation ba = annotation.annotationType().getAnnotation(BindingAnnotation.class);
                     try {
-                        BinderFactory fact = ba.value().newInstance();
-                        binders.add(new Bindifier(this.method, annotation, param_idx, (Binder<Annotation, Object>)fact.build(annotation)));
+                        BinderFactory<Annotation, Object> fact = (BinderFactory<Annotation, Object>) ba.value().newInstance();
+                        binders.add(new Bindifier<Annotation>(this.method, annotation, param_idx, fact.build(annotation)));
```

Comment:
The presence of a generic cast here suggests that `Bindifier` should expect a `BinderFactory<AnnoType extends Annotation, ?>`. Bindifier isn't using the second parameter.